app/assets/stylesheets/layout/_layout.scss

```diff
@@ -45,6 +45,17 @@ body {
       margin-bottom: 40px;
     }
   }
+
+}
+
+.registrations .box, .sessions .box{
+  width: 40%!important;
+  margin: 0 auto;
+
+  form input[type="submit"]{
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 4<br>Opening curly brace `{` should be preceded by one space